lib/hash.py

```diff
@@ -0,0 +1,46 @@
+import hashlib
```

Comment:
When I add new libs I always add a first line, comment describing what the file is. Here it would be something like `Hashing library (Part of the PixelToolkit project)`.